Project.toml

```
@@ -1,6 +1,6 @@
 name = "DataStructures"
 uuid = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
-version = "0.18.2"
+version = "0.18.3"
```

since this is a completely invisable change to the user, lets leave the version number as is

```suggestion
version = "0.18.2"
```